### PR TITLE
W4- tests added for linkCrawler

### PR DESCRIPTION
**Pull request by dev-siris**
I have added tests for linkcrawler 😃

### CODE REVIEW

**Comment by syuraj** [comment]
@chetannn tests seem to be failing

**Comment by dev-siris** [comment]
I have onlinekhabar test issue and added setopati link test too..

File: src/jobs/NewsCrawlerTrigger/__tests__/linkCrawler.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+describe('link-crawler integration test', () => {
+	it('link-crawler show crawl setopati link', async () => {
+		const baseUrl = 'https://setopati.com'
+		const url = 'https://setopati.com/sports'
+		const { error, links } = await scrapeNewsLink(baseUrl, url)
+		expect(links).not.toBe(null)
+		expect(error).toBeFalsy()
+	})
+	it('link-crawler show crawl setopati link', async () => {
```
Comment (syuraj) [inline src/jobs/NewsCrawlerTrigger/__tests__/linkCrawler.test.js]: @chetannn is it not same test as above?

**Comment by syuraj** [comment]
can you add azure work item no to commits as in title from now on? @dev-siris @chetannn

File: src/jobs/NewsCrawlerTrigger/__tests__/linkCrawler.test.js (hunk 1 above)
Comment (dev-siris) [reply src/jobs/NewsCrawlerTrigger/__tests__/linkCrawler.test.js]: sorry my mistake

**Comment by dev-siris** [comment]
new changes are pushed

**Pull request merged**